index.js

```diff
@@ -139,9 +151,41 @@ Glance.prototype.serveRequest = function Glance$serveRequest (req, res) {
 
 function showError (errorCode, req, res) {
   res.writeHead(errorCode, RESPONSE_HEADERS)
-  fs.createReadStream(
-      path.join(__dirname, 'errors', errorCode + '.html')
-  ).pipe(res)
+
+  var errorHtml = ''
+
+  var errorPage = fs.createReadStream(
+    path.join(__dirname, 'errors', errorCode + '.html')
+  )
+
+  errorPage.on('data', function (buf) {
+    errorHtml += buf.toString()
+  })
+
+  errorPage.on('end', function () {
+    var title = errorTitle(errorCode)
+    renderPage(title, errorHtml, res)
+  })
+}
+
+function renderPage (title, body, res) {
+  var layout = fs.createReadStream(
+    path.join(__dirname, 'errors/shared/layout.html')
+  )
+  layout
+  .pipe(replace(/{{\s*title\s*}}/g, title))
+  .pipe(replace(/{{\s*body\s*}}/g, body))
+  .pipe(res)
+}
+
+function errorTitle (errorCode) {
+  var mappings = {
+    '404': 'File Not Found',
+    '403': 'Forbidden',
+    '405': 'Method Not Allowed',
+    '500': 'Internal Server Error'
+  }
+  return mappings['' + errorCode]
```

Comment:
you shouldn't need the `'' + ` here

Reply:
Oh sure, I didn't remember/realize that toString couldn't be called on numeric literals but could on variables.  pushing a fix.